data_steward/analytics/table_metrics/weekly_scripts/erroneous_dates.py

```diff
@@ -0,0 +1,583 @@
+
+
+print('done.')
+# -
+
+cwd = os.getcwd()
+cwd = str(cwd)
+print("Current working directory is: {cwd}".format(cwd=cwd))
+
+# +
+dic = {
+    'src_hpo_id': [
+        "saou_uab_selma", "saou_uab_hunt", "saou_tul", "pitt_temple",
+        "saou_lsu", "trans_am_meyers", "trans_am_essentia", "saou_ummc",
+        "seec_miami", "seec_morehouse", "seec_emory", "uamc_banner", "pitt",
+        "nyc_cu", "ipmc_uic", "trans_am_spectrum", "tach_hfhs", "nec_bmc",
+        "cpmc_uci", "nec_phs", "nyc_cornell", "ipmc_nu", "nyc_hh",
+        "ipmc_uchicago", "aouw_mcri", "syhc", "cpmc_ceders", "seec_ufl",
+        "saou_uab", "trans_am_baylor", "cpmc_ucsd", "ecchc", "chci", "aouw_uwh",
+        "cpmc_usc", "hrhc", "ipmc_northshore", "chs", "cpmc_ucsf", "jhchc",
+        "aouw_mcw", "cpmc_ucd", "ipmc_rush", "va", "saou_umc"
+    ],
+    'HPO': [
+        "UAB Selma", "UAB Huntsville", "Tulane University", "Temple University",
+        "Louisiana State University",
+        "Reliant Medical Group (Meyers Primary Care)",
+        "Essentia Health Superior Clinic", "University of Mississippi",
+        "SouthEast Enrollment Center Miami",
+        "SouthEast Enrollment Center Morehouse",
+        "SouthEast Enrollment Center Emory", "Banner Health",
+        "University of Pittsburgh", "Columbia University Medical Center",
+        "University of Illinois Chicago", "Spectrum Health",
+        "Henry Ford Health System", "Boston Medical Center", "UC Irvine",
+        "Partners HealthCare", "Weill Cornell Medical Center",
+        "Northwestern Memorial Hospital", "Harlem Hospital",
+        "University of Chicago", "Marshfield Clinic",
+        "San Ysidro Health Center", "Cedars-Sinai", "University of Florida",
+        "University of Alabama at Birmingham", "Baylor", "UC San Diego",
+        "Eau Claire Cooperative Health Center", "Community Health Center, Inc.",
+        "UW Health (University of Wisconsin Madison)",
+        "University of Southern California", "HRHCare",
+        "NorthShore University Health System", "Cherokee Health Systems",
+        "UC San Francisco", "Jackson-Hinds CHC", "Medical College of Wisconsin",
+        "UC Davis", "Rush University", 
+        "United States Department of Veterans Affairs - Boston",
+        "University Medical Center (UA Tuscaloosa)"
+    ]
+}
+
+site_df = pd.DataFrame(data=dic)
+site_df
+
+# +
+######################################
+print('Getting the data from the database...')
+######################################
+
+site_map = pd.io.gbq.read_gbq('''
+    select distinct * from (
+    SELECT
+            DISTINCT(src_hpo_id) as src_hpo_id
+    FROM
+         `{DATASET}._mapping_visit_occurrence`
+         
+    UNION ALL
+    SELECT
+            DISTINCT(src_hpo_id) as src_hpo_id
+    FROM
+         `{DATASET}._mapping_care_site`
+         
+    UNION ALL
+    SELECT
+            DISTINCT(src_hpo_id) as src_hpo_id
+    FROM
+         `{DATASET}._mapping_condition_occurrence`  
```

Comment:
Jinja can be extremely helpful to generate this entire query from a 5 line template